src/youprop.h

```diff
@@ -27,7 +27,7 @@
 static long get_HFire_resistance() {
     return u.uprops[FIRE_RES].intrinsic;
 }
-static void set_HFire_resistance(hFireRes) {
+static void set_HFire_resistance(long hFireRes) {
```

Comment:
it didn't in cygwin with gcc. i think we need to remind clang to disallow implicit argument types.